### DISCUSSION TITLE
Prerequisists Question

### DISCUSSION THREAD

**Discussion by bossonic** [Axway-API-Management-Plus/apigateway-openlogging-elk#215] [Q&A]
Dear All,

As I need to install for private datacenter
Can I install on local machine with docker cluster? I asked this question because the specification is from AWS, this make me confuse a bit that's can install local docker cluster or not.

If can, what is specification of t2 largex2, because I cannot find out how large of server need to be for running this solutions if TPS is small.

Regards,
Nithi P.

**Comment by cwiechmann** [comment]
You can install the solution fully on-premise on any VM that has docker and docker-compose installed. 

The sizing of the infrastructure depends on the number of TPS in a 5 minute peak interval produced by all API-Gateways connected to the ELK-Environment. 
The reason is, that your servers must be able to handle incoming requests/events in real-time into Elasticsearch.
So, please let me know, what your maximum TPS for a 5 minute interval over the day is?

BTW: There is no such thing like docker cluster besides a Kubernetes platform.

Here find information about the EC2 instances:
https://aws.amazon.com/de/ec2/instance-types/t2/
However, these are only examples, as written in the documentation.
